lambdas/data-migration2/src/pdrs.ts

```diff
@@ -53,8 +53,13 @@ export const migratePdrRecord = async (
     }
   }
 
-  // Throw error if it was already migrated.
-  if (existingRecord) {
+  const isExistingRecordNewer = existingRecord
+    && existingRecord.updated_at >= new Date(dynamoRecord.updatedAt);
+
+  // Check the record's status here to prevent a granule from being written
+  // out-of-order. If we have already migrated this granule and the status
+  // of the new granule is 'running', skip it.
+  if (isExistingRecordNewer || (existingRecord && dynamoRecord.status === 'running')) {
```

Comment:
Same as above. PDRs also don't update if the state is `running`. The UPSERT logic is a little different than Granules though. The outcome is the same but there would need to be a different fix if we can't skip running records like this for some reason.

Reply:
Same question as above

Reply:
Same(ish) fix as above:

https://github.com/nasa/cumulus/pull/2162/commits/b57e0d8a7cb8e2b23ff7250774ab3fe3ee3e116a

We're not using the return of the upsert in `migratePdrRecord` the same way we do for Granules so this looks a little different. I still think it makes sense to throw the error here rather than further up the chain.